sormas-app/app/src/main/java/de/symeda/sormas/app/component/dialog/LocationDialog.java

```diff
@@ -288,6 +296,16 @@ public void setFacilityFieldsVisible(boolean visible, boolean clearOnHidden) {
 		}
 	}
 
+	public void setContinentFieldsVisible( boolean clearOnHidden) {
+		contentBinding.locationContinent.setVisibility(InfrastructureDaoHelper.loadContinents().isEmpty() ? GONE : VISIBLE);
```

Comment:
If I'm not mistaken, `loadContinents` and `loadSubcontinents` should always return false because you always add an empty item to the list, i.e. it should always contain at least one element. If that's the case, I'd rather directly call a backend method that counts the number of continents/subcontinents in the database here (also gets rid of the unnecessary logic that turns them into a list of items because those items aren't used here in the first place)